source/_components/sensor.sigfox.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Sigfox Sensor"
+description: "Display messages from Sigfox devices in Home Assistant."
+date: 2018-04-07 12:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: file.png
+ha_category: Sensor
+ha_iot_class: "Local Polling"
+ha_release: 0.68
+---
+
+
+[SigFox](https://www.sigfox.com/en) component adding a sensor for each Sigfox device registered with your account. The default name of sensors is `sigfox_{DEVICE_ID}` where `DEVICE_ID` is the devices Sigfox ID. The state of an added Sigfox sensor is the payload of the last message published by that device. Additionally there are attributes for the latitude and longitude coordinates of the device, as well as the signal-to-noise ratio ( [snr](https://en.wikipedia.org/wiki/Signal-to-noise_ratio)).
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: sigfox
+    api_login: your_api_login
+    api_password: your_api_password
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

Reply:
ok